src/Widgets/DeviceManagerWidget.vala

```diff
@@ -52,6 +54,29 @@ public class Sound.Widgets.DeviceManagerWidget : Gtk.Grid {
         update_showable ();
     }
 
+    private void disconnected () {
+        if (null == device_list) {
```

Comment:
This would normally be expressed `if (device_list == null) {`

Reply:
After I fixed the objects not correctly disposing, I found that I no longer needed the null check. I've removed it with the latest commit. Thanks!